validation/json-validator/src/main/java/com/redhat/mercury/util/schemavalidator/AbstructJsonSchemaValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.mercury.util.schemavalidator;
+
+import org.everit.json.schema.Schema;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+public abstract class AbstructJsonSchemaValidator {
+
+    protected Schema schema;
```

Comment:
declare it `private` and define a constructor receiving the Schema. Otherwise any subclass could just extend it and expect it to be created by default.

Reply:
I didn't exactly do what you said tell me what you think